doc/user/spec_examples.md

```diff
@@ -101,11 +102,52 @@ spec:
     backupIntervalInSecond: 300
     maxBackups: 5
     storageType: "S3"
+    s3:
+      s3Bucket: <S3-bucket-name>
+      awsSecret: <aws-secret-name>
+      prefix: <S3-prefix>
   restore:
     backupClusterName: "cluster-a"
     storageType: "S3"
 ```
 
+### Three members cluster that restores from different cluster's S3 backup
+
+etcd-operator can create a new cluster `cluster-b` from the backup of another cluster `cluster-a`. If backup exists, make sure the spec of `cluster-b` has the same `s3Bucket`, `awsSecret`, and `prefix` as those on the spec of `cluster-a`; so that the etcd operator can find the correct s3 backup path.
```

Comment:
make sure the backup policy of `cluster-b` has the same S3 configuration as those of `cluster-a`; etcd operator would also use that config to access snapshots of `cluster-a`.